Lib/diffenator/hbinput.py

```diff
@@ -36,10 +36,11 @@ def __init__(self, font):
         glyph_set = self.font.ttfont.getGlyphSet()
         for name in glyph_set.keys():
             glyph = glyph_set[name]
+            glyf = glyph_set.glyfTable[name]
```

Comment:
that'd only work for truetype, not for CFF, though..

Reply:
.. but also the previous code didn't work for CFF anyway

Reply:
Thanks. My solution also only works for the latest so I'm going to redo this.

Reply:
yeah. Just reread the readme. Clearly states TTF. so I'll press ahead and just worry about glyf.

Reply:
if you want the glyf table just get the glyf tabke fromn the font, as usual, don't get it from the glyph set object.
For CFF, you need to compute the control bounding box, charstrings have a `calcBounds` method that returns a 4-tuple

Reply:
yep, I'm now doing just `font['glyf'][name]` thanks.

Reply:
For diffenator2 we don't do any of this nonsense. Will be able to handle otf/ttf just fine. Just fixing this our CIs can work till diffenator2 is ready.